### PR TITLE
Graduate APIListChunking to GA documentation

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref https://github.com/kubernetes/enhancements/issues/365

/milestone 1.29

### CODE REVIEW

File: content/en/docs/reference/using-api/api-concepts.md [hunk 1]
```diff
@@ -316,17 +316,15 @@ The `content-encoding` header indicates that the response is compressed with `gz
...
 On large clusters, retrieving the collection of some resource types may result in
 very large responses that can impact the server and client. For instance, a cluster
 may have tens of thousands of Pods, each of which is equivalent to roughly 2 KiB of
 encoded JSON. Retrieving all pods across all namespaces may result in a very large
 response (10-20MB) and consume a large amount of server resources.
 
-Provided that you don't explicitly disable the `APIListChunking`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/), the
-Kubernetes API server supports the ability to break a single large collection request
+The Kubernetes API server supports the ability to break a single large collection request
```
Comment (sftim) [inline content/en/docs/reference/using-api/api-concepts.md:327]: Nice! Thanks.

**Comment by wojtek-t** [comment]
friendly ping - can I ask for approval?

**Pull request merged**